s127/models/information_type.py

```diff
@@ -298,6 +298,34 @@ class CategoryOfCommPref(models.TextChoices):
     # fmt: on
 
 
+# https://github.com/betagouv/SPPNautInterface/issues/244
+class FrequencyPair(s100.models.GenericComplexAttributeType):
```

Comment:
Do we want to keep this? I have a vague recollection that SHOM told us it wouldn't be used.